package.json

```diff
@@ -493,10 +499,15 @@
                     "group": "5_appServiceRemoteDebugCommands@2"
                 },
                 {
-                    "command": "appService.Refresh",
+                    "command": "appService.EnableFileLogging",
```

Comment:
So this would show up on every folder, even sub folders, correct? I feel like we really only need it on the "Log Files" node. Also the web app node - which you mentioned in the description, but I don't see here.

Reply:
Yeah, good point.  I think I'll create a new Subfolder type for all the other treeItems.  As far as being on the "Files" node, that was on Matt's request though I agree with you that it seems a bit excessive (though easier to have it on both).